packages/react/src/hooks/accounts/useConnect.ts

```diff
@@ -1,9 +1,9 @@
-import { useMutation } from '@tanstack/react-query'
 import { ConnectArgs, ConnectResult, connect } from '@wagmi/core'
 import * as React from 'react'
 
 import { useClient } from '../../context'
 import { MutationConfig } from '../../types'
+import { useMutation } from '../utils/query'
```

Comment:
Let's export `useMutation` from `src/hooks/utils` so we can import from `../utils` (like we do for `useQuery`).